### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by jfarrell468**
Set up a continuous build.

### CODE REVIEW

**Comment by gregbillock** [comment]
Why the leading dot on this file?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: cpp
+script: ./build_tests.sh && ./config_parser_test
+addons:
+  apt:
+    packages:
+    - libboost-all-dev
```
Comment (gregbillock) [inline .travis.yml]: shouldn't need boost for these config parser tests

**Comment by jfarrell468** [comment]
The leading dot is because it's a magic invisible file.

**Comment by gregbillock** [comment]
lgtm

**Pull request merged**